src/thermostat.js

```diff
@@ -173,7 +173,6 @@ export default function createThermostat({ Service, Characteristic }) {
         fn(...args)
         this.startDisconnectTimeout()
       }, (err) => {
-        err.isError = true
```

Comment:
which line exactly? 
(I can't get the line number out of the link you've provided :()